client/src/components/tempChartConfig.ts

```diff
@@ -0,0 +1,29 @@
+export const data = {
+  labels: [
+    'January',
+    '',
+    'March',
+    '',
+    'May',
+    '',
+    'July',
+    '',
+    'September',
+    '',
+    'November',
+    '',
+    'December'
+  ],
+  datasets: [
+    {
+      label: 'Data One',
```

Comment:
can this be removed since its done in another issue?

Reply:
This file was added to the branch when I merged from main and these changes were made automatically when I ran yarn format. Should I try and undo this change?